src/script/page/MainContent/panels/preferences/accountPreferences/AvailabilityButtons.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import {amplify} from 'amplify';
+import {WebAppEvents} from '@wireapp/webapp-events';
+import {Availability} from '@wireapp/protocol-messaging';
+import cx from 'classnames';
+
+import {t} from 'Util/LocalizerUtil';
+import Icon from 'Components/Icon';
+import {CSS_SQUARE} from 'Util/CSSMixin';
+import {CSSObject} from '@emotion/serialize';
+import {ContextMenuEntry} from '../../../../../ui/ContextMenu';
+
+interface AvailabilityInputProps {
+  availability: Availability.Type;
+}
+
+const iconStyles: CSSObject = {
+  ...CSS_SQUARE(10),
+  fill: 'currentColor',
+  margin: '0 6px 1px 0',
+  minWidth: 10,
+  stroke: 'currentColor',
+};
+
+const activeStyles: CSSObject = {
+  background: '#E7F0FA',
+  border: '1px solid #6AA4DE',
+  color: '#0667C8',
+};
+
+const headerStyles: CSSObject = {
+  lineHeight: '14px',
+  margin: '37px 0 6px',
+  padding: 0,
+  textTransform: 'uppercase',
+};
+
+const AvailabilityButtons: React.FC<AvailabilityInputProps> = ({availability}) => {
+  const icons: {
+    [key: string]: any;
+  } = {
+    [Availability.Type.AVAILABLE]: (
+      <Icon.AvailabilityAvailable
+        className="availability-state-icon"
+        css={{...iconStyles, fill: '#1D7833', stroke: '#1D7833'}}
```

Comment:
I think we should use `var(--red-500)` instead hardcoded value as a fill there. This will provide the correct colors in the dark mode.

Reply:
you are right - I planned to fix it in the separate PR because I think there is more things to be amended in this component